### PR TITLE
Feature/downloadable excel report for parent wdf report fe

### PR DESCRIPTION
**Pull request by mksrivastava**

### CODE REVIEW

File: src/app/core/services/report.service.ts [hunk 1]
```diff
@@ -41,4 +41,11 @@ export class ReportService {
       responseType: 'blob' as 'json',
     });
   }
+
+  getPatrentWDFReport(workplaceUid: string): Observable<HttpResponse<Blob>> {
```
Comment (pjshumphreys) [inline src/app/core/services/report.service.ts]: There's a typo here. Should be  getParentWDFReport right?

File: src/app/features/reports/pages/workplaces/workplaces.component.ts [hunk 2]
```diff
@@ -39,6 +51,24 @@ export class WorkplacesComponent implements OnInit {
     );
   }
 
+  public downloadWdfParentReport(event: Event) {
+    event.preventDefault();
+    this.subscriptions.add(
+      this.reportService
+        .getPatrentWDFReport(this.primaryWorkplace.uid)
```
Comment (pjshumphreys) [inline src/app/features/reports/pages/workplaces/workplaces.component.ts]: Typo here also  it seems

File: src/app/core/services/report.service.ts (hunk 1 above)
Comment (mksrivastava) [reply src/app/core/services/report.service.ts]: yes. Corrected. Thanks

File: src/app/features/reports/pages/workplaces/workplaces.component.ts (hunk 2 above)
Comment (mksrivastava) [reply src/app/features/reports/pages/workplaces/workplaces.component.ts]: yes. Corrected. Thanks

File: src/app/core/services/report.service.ts (hunk 1 above)
Comment (pjshumphreys) [reply src/app/core/services/report.service.ts]: thanks :)

**Pull request merged**